src/main/scala/microsites/util/MicrositeHelper.scala

```diff
@@ -38,13 +38,18 @@ class MicrositeHelper(config: MicrositeSettings) {
     copyPluginResources(pluginURL, s"$targetDir$jekyllDir/", "css")
     copyPluginResources(pluginURL, s"$targetDir$jekyllDir/", "img")
     copyPluginResources(pluginURL, s"$targetDir$jekyllDir/", "js")
+    copyPluginResources(pluginURL, s"$targetDir$jekyllDir/", "layout")
```

Comment:
Is the plugin providing any custom html layout? If it isn't, this line is not needed